runtime/redirector/redirector.c

```diff
@@ -535,8 +535,8 @@ chooseJVM(JavaVMInitArgs *args, char *retBuffer, size_t bufferLength)
 	char *xmxstr = NULL;
 	U_64 requestedHeapSize = 0;
 
-	/* the command line is handled below but look into the ENVVAR_IBM_JAVA_OPTIONS here, since it is a special case */
-	envOptions = getenv(ENVVAR_IBM_JAVA_OPTIONS);
+	/* the command line is handled below but look into the ENVVAR_OPENJ9_JAVA_VERSION here, since it is a special case */
+	envOptions = getenv(ENVVAR_OPENJ9_JAVA_VERSION);
```

Comment:
@pdbain-ibm To give IBM time to migrate from the IBM_JAVA_OPTIONS to the OPENJ9_JAVA_OPTIONS env variable, please fall back and `getenv(IBM_JAVA_OPTIONS)` if the openj9 version is null.